examples/python/agent-samples/youtube-shorts-assistant/agent.py

```diff
@@ -0,0 +1,52 @@
+# Shows how to call all the sub-agents using the LLM's reasoning ability. Run this with "adk run" or "adk web"
+
+from google.adk.agents import LlmAgent
+from google.adk.tools import google_search
+
+from util import load_instruction_from_file
+
+# --- Sub Agent 1: Scriptwriter ---
+scriptwriter_agent = LlmAgent(
+    name="ShortsScriptwriter",
+    model="gemini-2.0-flash-001",
+    instruction=load_instruction_from_file("scriptwriter_instruction.txt"),
```

Comment:
```suggestion
    instruction=open("scriptwriter_instruction.txt").read,
```

Reply:
Hey, I kept the try...except for the file read to handle exceptions gracefully. Prefer this for robustness unless you strongly disagree.